test/components/split/splitcomboview.coffee

```diff
@@ -0,0 +1,28 @@
+should = require 'should'
+sinon = require 'sinon'
+shouldSinon = require 'should-sinon'
+KDSplitComboView = require '../../../lib/components/split/splitcomboview'
+KDView = require '../../../lib/core/view'
+
+
+describe 'KDSplitComboView', ->
+  beforeEach ->
+    @sinon = sinon.sandbox.create()
+    @view = new KDView
+      tooltip :
+        title : 'hello'
+
+    @o = {}
+    @o.views = [@view, @view]
+    @instance = new KDSplitComboView @o, {}
```

Comment:
`data` should be optional, if this view is failing to initialize without a proper `data` we need to write down a test for that case as well.